### PR TITLE
chore: GoodReads Sample - Update to net6

### PR DESCRIPTION
**Pull request by aesteves900**
Closes https://github.com/unoplatform/Uno.Samples/issues/273

### CODE REVIEW

**Comment by nickrandolph** [comment]
@aesteves900 the build error is related to the global.json file that you have which is pinning the sdk to 5.x. Remove this and that should fix the build issue

image

File: UI/UnoGoodReads/UnoGoodReads/UnoGoodReads.Shared/MainPage.xaml [hunk 1]
```diff
@@ -1,50 +1,14 @@
...
-	  xmlns:pages="using:UnoGoodReads.Views"
-	  xmlns:vm="using:UnoGoodReads.ViewModels"
-	  mc:Ignorable="d"
-	  xmlns:utu="using:Uno.Toolkit.UI"
-	  xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
-	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-
-    <muxc:TabView x:Name="tabView"
-				  Margin="0,35">
-        <muxc:TabView.TabItems>
```
Comment (nickrandolph) [inline UI/UnoGoodReads/UnoGoodReads/UnoGoodReads.Shared/MainPage.xaml:17]: @aesteves900 why has all the content been removed?

**Comment by nickrandolph** [comment]
@aesteves900 moving this to draft - the sample doesn't work because the contents on MainPage has been removed

**Comment by aesteves900** [comment]
@nickrandolph , after the update I have an issue  on windows that occurs after a few seconds:

image

File: UI/UnoGoodReads/UnoGoodReads/UnoGoodReads.Shared/Styles/MainDictionary.xaml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 ﻿<ResourceDictionary xmlns:uen="using:Uno.Extensions.Navigation.UI" xmlns:uer="using:Uno.Extensions.Reactive.UI" xmlns:utu="using:Uno.Toolkit.UI" xmlns:muxc="using:Microsoft.UI.Xaml.Controls" xmlns:um="using:Uno.Material" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:d="http://schemas.microsoft.com/expression/blend/2008" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
-  <ResourceDictionary.ThemeDictionaries>
-    <ResourceDictionary x:Key="Light">
+  <ResourceDictionary.ThemeDictionaries>
```
Comment (nickrandolph) [inline UI/UnoGoodReads/UnoGoodReads/UnoGoodReads.Shared/Styles/MainDictionary.xaml:2]: @aesteves900 why did you remove the Light/Dark options in the ThemeDictionaries?

**Review by nickrandolph — APPROVED** [review]

**Comment by nickrandolph** [comment]
Referencing [this issue](https://github.com/unoplatform/Uno.Samples/issues/273) for adding missing targets

File: UI/UnoGoodReads/UnoGoodReads/UnoGoodReads.Shared/Styles/MainDictionary.xaml (hunk 2 above)
Comment (aesteves900) [reply UI/UnoGoodReads/UnoGoodReads/UnoGoodReads.Shared/Styles/MainDictionary.xaml]: Yes, I removed it because there is no switch button.
